src/main/resources/static/new.css

```diff
@@ -680,20 +680,24 @@ input.toDelegate:checked + span.weekIcon + span.weekIcon {
     border-radius: .3rem;
     padding: .666rem;
     transition: opacity 1s ease-out;
```

Comment:
add this 
![image](https://github.com/TDZ-production/todoz/assets/137875117/32ccfea6-7877-4db4-9d58-f8752e921086)

Reply:
okay opraveno, tedkon vypada takle: 
![Uploading image.png…]()